lab1/src/com/company/Сircle.java

```diff
@@ -0,0 +1,82 @@
+package com.company;
+
+import java.awt.*;
+import java.awt.geom.*;
+
+class Circle extends Canvas implements Shape{
+
+    private Ellipse2D circle;
+
+    private Color borderColor;
+    private Color backgroundColor;
+
+    public Circle(int centerX, int centerY, int radius, Color border, Color background) {
+        this.circle = new Ellipse2D.Double(centerX - radius, centerY - radius, 2 * radius, 2 * radius);

```

Comment:
maybe move this to var, something like

```suggestion
        int diameter = 2 * radius;
        this.circle = new Ellipse2D.Double(centerX - radius, centerY - radius, diameter, diameter);
```